chain/network/src/testonly/stream.rs

```diff
@@ -7,7 +7,7 @@ use crate::network_protocol::{Encoding, PeerMessage};
 use crate::tcp;
 
 pub struct Stream {
-    stream: tcp::Stream,
+    pub stream: tcp::Stream,
```

Comment:
I think you can make this `pub (crate)`. In general, aim to limit visibility to the lowest level needed.

Reply:
Cool, didn't know the differences but just learnt it here
- https://web.mit.edu/rust-lang_v1.25/arch/amd64_ubuntu1404/share/doc/rust/html/reference/visibility-and-privacy.html#pubin-path-pubcrate-pubsuper-and-pubself

updated to `pub (crate)` as suggested.